lib/phoenix/pubsub/local.ex

```diff
@@ -69,13 +71,43 @@ defmodule Phoenix.PubSub.Local do
       :ok
 
   """
+  def broadcast(local_server, from, topic, %Broadcast{event: event} = msg)
+    when is_atom(local_server) do
+
+    local_server
+    |> subscribers(topic)
+    |> Enum.reduce(%{}, fn
+      {pid, _fastlanes}, cache when pid == from ->  cache
+
+      {pid, nil}, cache ->
+        send(pid, msg)
+        cache
+
+      {pid, {fastlane_pid, serializer, event_intercepts}}, cache ->
+        if event in event_intercepts do
+          send(pid, msg)
+          cache
+        else
+          case Map.fetch(cache, serializer) do
+            {:ok, encoded_msg} ->
+              send(fastlane_pid, encoded_msg)
+              cache
+            :error ->
+              encoded_msg = serializer.encode!(msg)
```

Comment:
I was planning to call the callback `fastlane!`. The reason is that encoding is a special implenentation detail for fastlaning for transports. But it is not a requirement for all pubsub subscribers. Calling something like `fastlane!` wouldn't couple to our channel+transports system (at the costing of slightly augmenting the serializer API surface).